harness/machine_types/rgt_test.py

```diff
@@ -406,14 +406,14 @@ def get_walltime(self):
 
     def get_total_processes(self):
         val = self._get_builtin_param("total_processes")
```

Comment:
LIke the earlier ones, this seems correct to consider an empty string as an unset value.